web/src/enterprise/campaigns/detail/CampaignStatsCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ProgressCheckIcon from 'mdi-react/ProgressCheckIcon'
+import SourceMergeIcon from 'mdi-react/SourceMergeIcon'
+import SourceBranchIcon from 'mdi-react/SourceBranchIcon'
+import CheckCircleOutlineIcon from 'mdi-react/CheckCircleOutlineIcon'
+import classNames from 'classnames'
+import { CampaignFields } from '../../../graphql-operations'
+
+interface CampaignStatsCardProps extends Pick<CampaignFields['changesets'], 'stats'> {}
+
+export const CampaignStatsCard: React.FunctionComponent<CampaignStatsCardProps> = ({ stats }) => {
+    const percentDone = stats.total === 0 ? 0 : (((stats.closed + stats.merged) / stats.total) * 100).toFixed(0)
+    const isDone = stats.closed + stats.merged === stats.total
```

Comment:
Nitpicky nitpick: I'd use `complete` instead of `done`, since that's what it says in the UI too ("34% complete").

```suggestion
    const completion = stats.total === 0 ? 0 : (((stats.closed + stats.merged) / stats.total) * 100).toFixed(0)
    const completed = stats.closed + stats.merged === stats.total
```

Reply:
Good call, done